src/skin/legacy/tooltips.cpp

```diff
@@ -427,13 +427,41 @@ void Tooltips::addStandardTooltips() {
             << QString("%1: %2").arg(leftClick, tempoTapButton)
             << QString("%1: %2").arg(rightClick, bpmTapButton);
 
+    add("beats_set_change_marker")
+            << tr("Set Beat Marker")
+            << tr("Set a beat marker at the current play position.");
+
+    add("beats_remove_marker")
+            << tr("Remove Beat Marker")
+            << tr("Remove the beat marker at the current play position.");
+
+    add("beats_increase_bar_length")
+            << tr("Adjust the bar length")
+            << tr("Increment by one the bar length of the current marker.");
```

Comment:
```suggestion
            << tr("Increment bar lenght  by one beat, between adjacent markers.");
```
The difficult part is what to do if a padding bar is created? I can imagine to set a "temporary'" beat change marker, at the beginning of the padding bar. The user can then check afterwards where bar Signatur actually ends and remove one f the Change markes.